hs_core/templates/pages/my-resources.html

```diff
@@ -56,15 +56,19 @@ <h2 class="page-title">Resources</h2>
             </table>
 
             <div class="btn-toolbar">
-                <a class="btn btn-link" href="?start=0"><i class="fa fa-fast-backward"></i></a>
-                <a class="btn btn-link" href="?start={{ first|add:"-20" }}"><i class="fa fa-backward"></i></a>
+                <a {% if first == 0 %} disabled {% endif %}
+                        class="btn btn-link" href="?start=0"><i class="fa fa-fast-backward"></i></a>
+                <a {% if first == 0 %} disabled {% endif %}
+                        class="btn btn-link" href="?start={{ first|add:"-20" }}"><i class="fa fa-backward"></i></a>
                 {% if last %}
                     Results {{ first|add:"1" }}-{{ last }} of {{ ct }}
                 {% else %}
                     Results 0-0 of 0
                 {% endif %}
-                <a class="btn btn-link" href="?start={{ first|add:"20" }}"><i class="fa fa-forward"></i></a>
-                <a class="btn btn-link" href="?start={{ ct|add:"-20" }}"><i class="fa fa-fast-forward"></i></a>
+                <a {% if last == ct %} disabled {% endif %}
```

Comment:
The value of 'last' should never be more than total number of resources (ct). Otherwise, the same problem as I mentioned above.